db/models/user.js

```diff
@@ -2,27 +2,48 @@
 
 // bcrypt docs: https://www.npmjs.com/package/bcrypt
 const bcrypt = require('bcryptjs')
-    , {STRING, VIRTUAL} = require('sequelize')
+    , {BOOLEAN, STRING, VIRTUAL} = require('sequelize')
 
 module.exports = db => db.define('users', {
-  name: STRING,
+  userName: STRING,
   email: {
     type: STRING,
     validate: {
       isEmail: true,
-      notEmpty: true,
     }
   },
+  firstName: STRING,
+  lastName: STRING,
+  shippingAddress: STRING ,
```

Comment:
Addresses are hard to standardize, especially if considering international shipping. We'll want to consider standards that give separate fields for each line -- check out this StackOverflow http://stackoverflow.com/questions/310540/best-practices-for-storing-postal-addresses-in-a-database-rdbms